packages/react-table/src/components/Table/utils/decorators/selectable.tsx

```diff
@@ -9,7 +9,7 @@ export const selectable: ITransform = (
   { rowIndex, columnIndex, rowData, column, property }: IExtra
 ) => {
   const {
-    extraParams: { onSelect, allRowsSelected, rowLabeledBy = 'simple-node' }
+    extraParams: { onSelect, allRowsSelected, selectedRowsAmount, rowLabeledBy = 'simple-node' }
```

Comment:
If we no longer need `allRowsSelected`, can we remove it?

Reply:
I'll remove it, I kept it in in case something depended on it.